src/main/java/com/hazelcast/eurekast/one/EurekastOneProperties.java

```diff
@@ -26,7 +26,7 @@
  * <p>Configuration class of the Hazelcast Discovery Plugin for <a href="https://github.com/Netflix/eureka">Eureka 1</a>.</p>
  * <p>For possible configuration properties please refer to the public constants of this class.</p>
  */
-public final class EurekastOneProperties {
+final class EurekastOneProperties {
```

Comment:
The idea is to have those classes public to generate Javadoc for it. Javadoc can act as the documentation of available properties in this case. It was on purpose :)